package.json

```diff
@@ -29,6 +29,7 @@
     "dev": "tsc -w",
     "lint": "eslint src/",
     "test": "jest src/ --collectCoverage",
+    "test-watch": "jest src/ --watch",
```

Comment:
I'm happy to switch to watch, just remove the watch all script too. Currently there are 2 `test-watch`

Reply:
🤦  sorry, that was a dumb mistake 😆 
will fix tonight :)

Reply:
S'all good, and again, a massive thanks for the PR.

Reply:
fixed it ✅